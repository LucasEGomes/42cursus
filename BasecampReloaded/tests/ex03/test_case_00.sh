#!/bin/sh
mkdir -p temp
touch temp/file{1..3}.sh
cp $ROOT_DIR/ex03/find_sh.sh temp/
cd temp
bash find_sh.sh | sort | cat -e > ../test_case_00.user
cd ..
diff test_case_00.expected test_case_00.user > test_case_00.output
if [ -z "$(cat test_case_00.output)" ];
then
    echo -n "O" >> $TEST_SUMMARY
else
    echo -n "X" >> $TEST_SUMMARY
fi
rm -rf temp/
