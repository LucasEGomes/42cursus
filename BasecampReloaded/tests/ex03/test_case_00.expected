file1$
file2$
file3$
find_sh$
